components/konflux-ui/staging/base/proxy/nginx.conf

```diff
@@ -161,6 +161,24 @@ http {
             include /mnt/nginx-generated-config/bearer.conf;
         }
 
+        location ~* /api/v1/namespaces(/?)$ {
+            # authenticate request
+            auth_request_set $email  $upstream_http_x_auth_request_email;
+            auth_request /oauth2/auth;
+
+            # Impersonate requesting user
+            proxy_set_header Impersonate-User $email;
+            include /mnt/nginx-generated-config/bearer.conf;
+
+            # namespace-lister endpoint
+            rewrite ^/(.*)/$ /$1 permanent;
```

Comment:
we should use `break` instead of `permanent`. we want to reverse proxy not redirect.